addons/point_of_sale/static/src/app/models/pos_order_line.js

```diff
@@ -366,11 +366,23 @@ export class PosOrderline extends Base {
     }
 
     get unitDisplayPrice() {
-        if (this.config.iface_tax_included === "total") {
-            return this.allUnitPrices.priceWithTax;
-        } else {
-            return this.allUnitPrices.priceWithoutTax;
-        }
+        const prices =
+            this.combo_line_ids.length > 0
+                ? this.combo_line_ids.reduce(
+                      (acc, cl) => {
+                          const clPrices = cl.getAllPrices(1);
+                          return {
+                              priceWithTax: acc.priceWithTax + clPrices.priceWithTax,
+                              priceWithoutTax: acc.priceWithoutTax + clPrices.priceWithoutTax,
+                          };
+                      },
+                      { priceWithTax: 0, priceWithoutTax: 0 }
+                  )
+                : this.getAllPrices(1);
```

Comment:
```suggestion
                : this.allUnitPrices;
```

Reply:
`allUnitPrices` is cached when called, so calling it is better than calling `getAllPrices(1)` as if the value did not changed, we should not recompute it